### DISCUSSION TITLE
LMS/XMSS support

### DISCUSSION THREAD

**Discussion by derstrand** [open-quantum-safe/liboqs#1368] [Ideas]
I am curious if LMS/XMSS support is planned? Being that there is quite some demand for reasons of time being around, cryptanalysis, and respective existing standards. LMS/XMSS is also suggested by the CNSA 2.0 algorithm suite. Is this planned in the near future?

**Comment by baentsch** [comment]
Yes. An [initial PR for XMSS](https://github.com/open-quantum-safe/liboqs/pull/1275) is available but is in need of review/improvement. In addition, @ashman-p indicated his intention to work on an LMS integration (see https://github.com/orgs/open-quantum-safe/discussions/1346#discussioncomment-4657381). Helping hands welcome.
